tests/test_tokenization_common.py

```diff
@@ -368,11 +369,21 @@ def test_sentencepiece_tokenize_and_convert_tokens_to_string(self):
         # check if converting back to original text works
         reverse_text = tokenizer.convert_tokens_to_string(tokens)
 
-        if self.test_sentencepiece_ignore_case:
-            reverse_text = reverse_text.lower()
+        # All tokenizers pass this test without the below commented out code.
+        # if self.test_sentencepiece_ignore_case:
+        #     reverse_text = reverse_text.lower()
 
         self.assertEqual(reverse_text, text)
 
+        if isinstance(tokenizer, LayoutXLMTokenizer):
+            return
```

Comment:
Just for consistency with the rest of the code, when a `test_tokenization_common.py` test does not apply to a particular tokenizer, what we do is override that test in its test file and comment out why the test does not apply to it.

See for example what has been done for ByT5:

https://github.com/huggingface/transformers/blob/35ecf99cc4e1046e5908f5933a4548e43a0f4f1d/tests/byt5/test_tokenization_byt5.py#L309-L311

Would you mind making this change? :relaxed:

Reply:
Gladly!

Reply:
Thank you!!